addons/survey/models/survey_question.py

```diff
@@ -171,6 +172,12 @@ def default_get(self, fields):
             'All "Is a scored question = True" and "Question Type: Date" questions need an answer')
     ]
 
+    @api.depends('question_type')
+    def _compute_placeholder(self):
+        for question in self:
+            if question.question_type in ('simple_choice', 'multiple_choice', 'matrix'):
```

Comment:
`if question.question_type in ('simple_choice', 'multiple_choice', 'matrix') or not question.placeholder:`

To avoid CacheMiss errors.